qlib/contrib/evaluate.py

```diff
@@ -61,10 +61,16 @@ def cal_risk_analysis_scaler(freq):
         warnings.warn("risk_analysis freq will be ignored")
     if N is None:
         N = cal_risk_analysis_scaler(freq)
+        
+    if accumulation_mode not in ["summation", "exponential"]:
+        raise ValueError("Invalid value for `accumulation_mode`. Only 'summation' and 'exponential' are supported")
 
     mean = r.mean()
     std = r.std(ddof=1)
-    annualized_return = mean * N
+    if accumulation_mode == "summation":
+        annualized_return = mean * N
+    else:
+        annualized_return = (np.prod(1 + r) - 1)
     information_ratio = mean / std * np.sqrt(N)
```

Comment:
I'm not  sure if the annualized volatility should be chagned  if production is considered.
But intuitively, the accumulated volatility and production vollatility is different.